controller/rest/admwebhook.go

```diff
@@ -1141,11 +1141,13 @@ func (whsvr *WebhookServer) validate(ar *admissionv1beta1.AdmissionReview, globa
 	totalContainers := 0
 	{
 		reqImages := make([]string, 0, len(admResObject.AllContainers))
-		for _, containers := range admResObject.AllContainers {
-			for _, c := range containers {
-				reqImages = append(reqImages, c.Image)
+		if admResObject != nil {
```

Comment:
should lint 1144 be moved to line 1142?

Reply:
you're right, I've made the change.  thanks.